Tests/BackendIntegrationTests/Helpers/BaseStoreKitIntegrationTests+Verification.swift

```diff
@@ -26,7 +26,8 @@ extension BaseStoreKitIntegrationTests {
     @discardableResult
     func verifyEntitlementWentThrough(
         _ customerInfo: CustomerInfo,
-        file: FileString = #file,
+        file: FileString = #filePath,
+        filename: StaticString = #file,
```

Comment:
Whereas this seems like duplication, the reason I kept these 2 parameters is to preserve the existing behavior.
Before: Nimble's `FileString` was a typealias for `StaticString`.
Now: v13.7.1 Nimble's `FileString` is a typealias for `String`, which does not match anymore with the `StaticString` that `XCTUnwrap` (in line 60) expects.

And I did `file: FileString = #filePath` instead of `file: FileString = #file` because that's the actual default value for `file` in Nimble now.
*